tfhe/benches/core_crypto/pbs128_bench.rs

```diff
@@ -1,109 +1,160 @@
+    let lwe_noise_distribution = DynamicDistribution::new_t_uniform(45);
+    let glwe_noise_distribution = DynamicDistribution::new_t_uniform(24);
+    let pbs_base_log = DecompositionBaseLog(24);
+    let pbs_level = DecompositionLevelCount(3);
+    let ciphertext_modulus = CiphertextModulus::new_native();
+
+    let params_name = "PARAMS_SWITCH_SQUASH";
+
+    let mut boxed_seeder = new_seeder();
+    let seeder = boxed_seeder.as_mut();
+
+    let mut secret_generator =
+        SecretRandomGenerator::<ActivatedRandomGenerator>::new(seeder.seed());
+
+    let mut encryption_generator =
+        EncryptionRandomGenerator::<ActivatedRandomGenerator>::new(seeder.seed(), seeder);
+
+    let input_lwe_secret_key =
+        LweSecretKey::generate_new_binary(lwe_dimension, &mut secret_generator);
+
+    let output_glwe_secret_key = GlweSecretKey::<Vec<Scalar>>::generate_new_binary(
+        glwe_dimension,
+        polynomial_size,
+        &mut secret_generator,
+    );
+
+    let output_lwe_secret_key = output_glwe_secret_key.clone().into_lwe_secret_key();
+
+    let mut bsk = LweBootstrapKey::new(
+        Scalar::ZERO,
+        glwe_dimension.to_glwe_size(),
+        polynomial_size,
+        pbs_base_log,
+        pbs_level,
+        lwe_dimension,
+        ciphertext_modulus,
+    );
+    par_generate_lwe_bootstrap_key(
+        &input_lwe_secret_key,
+        &output_glwe_secret_key,
+        &mut bsk,
+        glwe_noise_distribution,
+        &mut encryption_generator,
+    );
+
+    let mut fourier_bsk = Fourier128LweBootstrapKey::new(
+        lwe_dimension,
+        glwe_dimension.to_glwe_size(),
+        polynomial_size,
+        pbs_base_log,
+        pbs_level,
+    );
+    convert_standard_lwe_bootstrap_key_to_fourier_128(&bsk, &mut fourier_bsk);
+
+    let message_modulus: Scalar = 1 << 4;
+
+    let input_message: Scalar = 3;
+
+    let delta: Scalar = (1 << (Scalar::BITS - 1)) / message_modulus;
+
+    let plaintext = Plaintext(input_message * delta);
+
+    let lwe_ciphertext_in: LweCiphertextOwned<Scalar> = allocate_and_encrypt_new_lwe_ciphertext(
+        &input_lwe_secret_key,
+        plaintext,
+        lwe_noise_distribution,
+        ciphertext_modulus,
+        &mut encryption_generator,
+    );
+
+    let accumulator: GlweCiphertextOwned<Scalar> = GlweCiphertextOwned::new(
+        Scalar::ONE,
+        glwe_dimension.to_glwe_size(),
+        polynomial_size,
+        ciphertext_modulus,
+    );
+
+    let mut out_pbs_ct: LweCiphertext<Vec<Scalar>> = LweCiphertext::new(
+        0,
+        output_lwe_secret_key.lwe_dimension().to_lwe_size(),
+        ciphertext_modulus,
+    );
+
+    let fft = Fft128::new(polynomial_size);
+    let fft = fft.as_view();
+
+    let mut buffers = vec![
+        0u8;
+        bootstrap_scratch::<Scalar>(
+            fourier_bsk.glwe_size(),
+            fourier_bsk.polynomial_size(),
+            fft
+        )
+        .unwrap()
+        .unaligned_bytes_required()
+    ];
+
```

Comment:
this will kill the performance, you can use

```rust
use crate::core_crypto::commons::computation_buffers::ComputationBuffers;

    let mut buffers = ComputationBuffers::new();
    buffers.resize(
        programmable_bootstrap_f128_lwe_ciphertext_mem_optimized_requirement::<Scalar>(
            fourier_bsk.glwe_size(),
            fourier_bsk.polynomial_size(),
            fft,
        )
        .unwrap()
        .unaligned_bytes_required(),
    );

    let stack = buffers.stack();
```

and pass the stack instead of the new podstack

Reply:
hmm or maybe not 🤔

Reply:
actually it's probably ok as is